site/docs/master/deploy-options.md

```diff
@@ -200,7 +200,7 @@ Now you can point your browser at the specified port on any node in your cluster
 
 You can run Contour without a Kubernetes Service at all.
 This is done by having the Contour pod run with host networking.
-Do this with `hostNetwork: true` on your pod definition.
+Do this with `hostNetwork: true` and `dnsPolicy: ClusterFirstWithHostNet` on your pod definition.
 Envoy will listen directly on port 8080 on each host that it is running.
 This is best paired with a DaemonSet (perhaps paired with Node affinity) to ensure that a single instance of Contour runs on each Node.
```

Comment:
Contour runs as a deployment and Envoy runs as a daemonset. Could you update this to reflect the changes from the old deployment model.